src/incipit/processing.py

```diff
@@ -128,6 +128,35 @@ def split_regions_by_image(
     return regions_by_image_index
 
 
+def adjust_region_position_by_image(
+        regions: typing.List[typing.Union[Region, IndexedRegion]],
+        expand_vertical_ratio_of_system: float = 0.5,
+        expand_vertical_ratio_of_gap: float = 0.45,
+)-> typing.List[typing.Union[Region, IndexedRegion]]:
+    leftmost_system_boundary = sys.maxsize
+    rightmost_system_boundary = 0
+    previous_system_bottom_y_coordinate = 0
+    index = 0
+    min_vertical_gap_between_systems = sys.maxsize
+    for region in regions:
```

Comment:
@Allan518 I would put a comment above this loop, like:
```
# In first pass of regions, we detect leftmost and rightmost boundaries
```

Reply:
@jlumbroso I have made the change, please review.